UefiCpuPkg/Test/Mock/Include/GoogleTest/Library/MockTimerLib.h

```diff
@@ -0,0 +1,29 @@
+/** @file MockTimerLib.h

```

Comment:
Can you split the three tests into separate commits and mark your PR as [REBASE & FF]? This makes it easier for us to take directly to edk2.

Reply:
Checked in the code with 3 separate commits and Title updated as suggested